src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -5100,6 +5100,29 @@ def test_aks_stop_and_start(self, resource_group, resource_group_location):
         start_cmd = 'aks start --resource-group={resource_group} --name={name}'
         self.cmd(start_cmd)
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_stop_and_start_private_cluster(self, resource_group, resource_group_location):
+        # This is the only test case in which the `--generate-ssh-keys` option is enabled in the `aks create` command.
+        # Please do not enable this option in other test cases to avoid race conditions during concurrent testing.
+        # For more details, please refer to issue #18854.
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --generate-ssh-keys --enable-private-cluster'
```

Comment:
please don't generate a new ssh key in this test case, use an existing one instead (see example in the test case below)

Reply:
fixed